docs/sql_reference/src/asciidoc/_chapters/sql_utilities.adoc

```diff
@@ -48,6 +49,326 @@ to connect to and issue SQL utilities, see the
 {docs-url}/client_installation/index.html[_{project-name} Client Installation Guide_].
 
 <<<
+[[cleanup_statement]]
+== CLEANUP Statement
+
+The CLEANUP Statement cleans up objects whose information is inconsistent in the metadata. This command can be used to clean up individual objects or all objects in a schema. This command can also be used to locate all inconsistent objects in the Trafodion metadata and then clean them up.
+
+Inconsistent objects can be cleaned up and removed from the metadata and HBase by using one of the following options.
+
+```
+CLEANUP {obj-type} {obj-name} [ ,UID {object-uid} ]

```

Comment:
Thanks Dave.

Could I change to:
`CLEANUP object-type object-name [ , UID {object-uid} ]` 
?

If so, the syntax of GRANT Statement should be changed to:
`....`
`TO grantee`
`...`

(delete the curly braces around `grantee` since just one choice?